CMake/platforms/windows.cmake

```diff
@@ -10,6 +10,8 @@ list(APPEND DEVILUTIONX_PLATFORM_LINK_LIBRARIES
   wininet
 )
 
+add_definitions(-DWINVER=0x0601 -D_WIN32_WINDOWS=0x0601 -D_WIN32_WINNT=0x0601)

```

Comment:
We also do this in windows9x.cmake, hopefully this doesn't break it (don't see why it should)

Reply:
When both (windows and windows9x) would be added, it would break. Only the first definition stays (at least in MSVC; I tested it).

That's why I changed `Platforms.cmake` to only include "windows.cmake" or "windows9x.cmake".
As far as I have seen "windows9x.cmake" includes everything that is also in "windows.cmake" + some windows 9x specific changes.

Reply:
Ah, makes sense, thanks!
These days treating windows 9x as a completely separate platform from modern Windows is sensible